### ISSUE TITLE
ESP32-2432S028R (Cheap Yellow Display) support

### ISSUE THREAD

**Issue by killabeenl** [DustinWatts/FreeTouchDeck#123]
Is it possible to add the ESP32-2432S028R (Cheap Yellow Display) to the web installer?
It is a nice all in one display with an esp32. But the display won't work with the standard installer

**Comment by DustinWatts** [comment]
Hey Killabee. I know the display. But the code for it is not mine. So I can just "steal" the binary file for the web installer. I'll have to talk to the person who forked the original library.

What code are you talking about?

**Comment by killabeenl** [comment]
Hello Dustin,

Thanks for the response. I tried to flash the device with: https://install.freetouchdeck.com/
I choose the ESP32 DevKitC option en flash the device. That is working.
But with the flash is complete the display does not work.

I also tried the manual install with the Arduino IDE software but got some errors there. I did some reading in your Discord channel and found that I need an older version of Arduino IDE.

Will try a manual installation tonight.
